packages/celer-web-app/src/ui/components/DocLineComponent/DocLineTextComponent.tsx

```diff
@@ -137,21 +155,76 @@ const StepNumberWithIcon: React.FC<DocLineTextWithIconProps> = ({docLine})=>{
 const Notes: React.FC<DocLineTextProps | DocLineTextWithIconProps> = ({docLine, altNotesColor})=>{
 	const {notes, variables} = docLine;
 	const styles = useStyles();
+
+	// If there are no notes for this step, return null
 	if(!notes){
 		return null;
 	}
 
+	// Otherwise, return as a standard note block
 	return  (
 		<div className={clsx(styles.notes, altNotesColor && styles.notesAlt)}>
 			<TypedStringComponent content={notes} variables={variables} isNotes/>
 		</div>
 	);
 };
 
+const NotesCollapsed: React.FC<DocLineTextCollapsedProps | DocLineTextWithIconCollapsedProps> = ({docLine, altNotesColor, bannerOpen, toggleBanner})=>{
+	const {notes} = docLine;
+	const styles = useStyles();
+
+	if (!notes) {
+		return (
+			<div className={clsx(styles.notes, styles.notesCollapsed, styles.notesCollapsedEmpty)}>
+				<span></span>
+			</div>         
+		);
+	}
+
+    const {setMapCenter} = useAppState();
+	return (
+		<div className={clsx(styles.notes, styles.notesCollapsed, altNotesColor && styles.notesAlt)}>
+			<div>
+				<span onClick = {() => {
+                    // If the banner is being opened, recenter the map
+                    if (!bannerOpen && docLine.centerCoord) {
+                        setMapCenter(docLine.centerCoord);
+                    }
+					toggleBanner(!bannerOpen);
+				}}>•••</span>
```

Comment:
nit: I would recommend using escapes for non-ascii characters in the source code